ArduCopter/mode_acro_heli.cpp

```diff
@@ -45,26 +45,29 @@ void ModeAcro_Heli::run()
     switch (motors->get_spool_state()) {
     case AP_Motors::SpoolState::SHUT_DOWN:
         // Motors Stopped
-        attitude_control->reset_target_and_rate(false);
```

Comment:
Changing this from false to true here will remove all counter talk on the tail rotor. If you can be still spinning down in this phase of flight this will let the tail move more than it would do now.

Low risk but something to consider.

Reply:
@lthall What do you mean by counter talk?  I think the issue here is that when disarmed, there won't be any target reset that would cause an issue with trying to fly disarmed.  I know this sounds crazy but I believe that this was intentional by my predecessor to allow users to fly in acro if the aircraft disarms unexpectedly, like due to code bug.  I think that this should true so that the attitude errors are reset.  However I may make this change in a separate PR to make this more visible as an intentional change.

Reply:
@Ferruccio1984 What do you think of having the attitude error correction (pitch and roll) removed for the disarmed state in acro.  This means that when you move the pitch and roll stick that the swashplate will not react while disarmed which is what you probably saw in your flight today.  I think it is a good idea because it would keep users from moving the swashplate while disarmed and then being surprised on spool up if the swash was not centered.  I think the idea of ever having to fly in a disarmed state is very low risk.

Reply:
I personally like that we can do "control surface" checks when on the ground.  Making sure that all of the servos and swash are moving and in the correct direction is a good pre-flight check.

Reply:
@MattKear i think my comment was not completely accurate. In acro currently, if you move the sticks, the  commanded attitude is tracked and you could end up having 90 deg commanded while on the ground and disarmed. With ths new implementation, you should still see movement of rhe swash because you are commanding rates and it will always center itself when you center the stick (zero rates) where it does not do that currently.  So you can stick check controls on the ground while disarmed. Again this is only in acro. Stabilize should not change except for yaw and that should react to rate requests also.